NorthstarDLL/audio.cpp

```diff
@@ -324,7 +324,7 @@ void CustomAudioManager::ClearAudioOverrides()
 
 	// slightly (very) bad
 	// wait for all audio reads to complete so we don't kill preexisting audio buffers as we're writing to them
-	std::unique_lock lock(g_CustomAudioManager.m_loadingMutex);
+	std::unique_lock lock(m_loadingMutex);
```

Comment:
Wait that changes sth else that wasn't [the formatting issue](https://github.com/R2Northstar/NorthstarLauncher/actions/runs/3208788455/jobs/5244955961#step:4:20) yet now formatting check passes? wtf?